test/jdk/sun/net/www/http/KeepAliveCache/B5045306.java

```diff
@@ -200,9 +199,8 @@ public void handle(HttpExchange trans) {
                     responseBody[i] = 0x41;
                 // override the Content-length header to be greater than the actual response body
                 trans.sendResponseHeaders(200, responseBody.length+1);
-                try(PrintWriter pw = new PrintWriter(trans.getResponseBody(), false, Charset.forName("UTF-8"))) {
-                    pw.print(responseBody);
-                }
+                OutputStream os = trans.getResponseBody();
+                os.write(responseBody);
```

Comment:
Hello Daniel, with this change, the OutputStream will no longer be explicitly closed before the exchange `trans` is closed. But I think that should be OK, because looking at the original code before https://github.com/openjdk/jdk/commit/95ca94436d12974d98b1b999f9cc8408d64cbe3c, it too wasn't closing it explicitly, as far as I can see it. So, this looks fine to me.

Reply:
Added a comment stating that we don't want to close the stream here.